apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/ProjectsResource.java

```diff
@@ -191,6 +192,20 @@ public Response retrieveProject(
         return Response.ok().entity(project).build();
     }
 
+    @POST
+    @Path("/delete")
+    @Operation(operationId = "deleteProjectsBatch", summary = "Delete projects", description = "Delete projects batch", responses = {
+            @ApiResponse(responseCode = "204", description = "No Content"),
+    })
+    public Response deleteProjectsBatch(
+            @RequestBody(content = @Content(schema = @Schema(implementation = BatchDelete.class))) @Valid BatchDelete batchDelete) {
+        String workspaceId = requestContext.get().getWorkspaceId();
+        log.info("Deleting projects by ids '{}', on workspace_id '{}'", batchDelete.ids(), workspaceId);
+        projectService.delete(batchDelete.ids());
```

Comment:
Yes, TBH I noticed that when deleting traces, we delete related spans, etc. However, I noticed that for a single project deletion we don't delete any children and assumed this is intended.
This way or another, I think this PR doesn't introduce any regression (since this is how a single project deletion is implemented).
If we decide to remove project's children we should probably do it in a separate PR after a proper discussion.

Reply:
I agree, but can we discuss this in the channel? My concern is that  with the bulk delete, this can happen much more frequently

Reply:
Sure, I'll write in the channel